netjsonconfig/schema.py

```diff
@@ -443,23 +443,39 @@
                 }
             }
         },
+        "encryption_mfp_property": {
+            "properties": {
+                "ieee80211w": {
+                    "type": "string",
+                    "title": "management frame protection",
+                    "enum": ["0", "1", "2"],
+                    "options": {"enum_titles": ["disabled", "optional", "required"]},
+                    "propertyOrder": 4,
+                }
+            }
+        },
```

Comment:
> For that reason I think we should keep this value as is and just introduce
> the new WPA personal mixed mode, maybe we can call the new option wpa_personal_sae_mixed?

Ok, I will leave the old options as they are. How about like this ?

| Description | value | old/new |
| ------------- | ------------- | ------------- |
| WPA Personal | wpa_personal | |
| WPA2 Personal | wpa2_personal | |
| WPA3 Personal | wpa3_personal | new |
| WPA Enterprise | wpa_enterprise | |
| WPA2 Enterprise | wpa2_enterprise | |
| WPA3 Enterprise | wpa3_enterprise | new |
| WPA/WPA2 Personal Mixed Mode | wpa_personal_mixed | |
| WPA2/WPA3 Personal Mixed Mode | wpa2_personal_mixed | new |
| WPA/WPA2 Enterprise Mixed Mode | wpa_enterprise_mixed | |
| WPA2/WPA3 Enterprise Mixed Mode | wpa2_enterprise_mixed | new |


> I think we will need a new encryption_mfp_property section which would be used only by WPA3 enterprise

Since 802.11w has existed before WPA3 and has been used together with WPA/WPA2, I think it should be able to be used for other than WPA3.

Reply:
@masap check the comment below (https://github.com/openwisp/netjsonconfig/pull/200#discussion_r787106873). The current schema for `encryption_mfp_property` will stay and it will be used with WPA2/WPA (allowing users . For WPA3, we will use  `encryption_mfp_property_required` which will set MFP to required by default.

Reply:
Sorry, I mis-understood. Ok, I will separate WPA3 only mode and use ```encryption_mfp_property_required```.

Reply:
@masap sounds great, I think we could show WPA3 first in the UI, then WPA2 and WPA, what do you think?

Reply:
I think it is nice. I will fix it like this.

| title |
| ------------- |
| No encryption |
| WPA3 Personal (new) |
| WPA3 Enterprise (new) |
| WPA2/WPA Personal |
| WPA2/WPA Enterprise |
| WPS |
| WEP |

Reply:
I fixed it like this. Because WPA3/WPA2/WPA Personal/Enterprise includes WPA3/WPA2 Personal/Enterprise Mixed Mode.

| title |
| ------------- |
| No encryption |
| WPA3 Personal (new) |
| WPA3 Enterprise (new) |
| WPA3/WPA2/WPA Personal (modified) |
| WPA3/WPA2/WPA Enterprise (modified) |
| WPS |
| WEP |